packages/core/src/actions/contracts/multicall.ts

```diff
@@ -93,6 +94,18 @@ export async function multicall<Data extends any[] = Result[]>({
     const { addressOrName, contractInterface, functionName } = <
       MulticallContract
     >contracts[i]
+
+    if (returnData === '0x') {
+      const err = new ContractMethodNoDataError({
+        addressOrName,
+        blockExplorer: chain.blockExplorers?.default,
+        functionName,
+      })
+      if (!allowFailure) throw err
+      console.warn(err.message)
```

Comment:
thought it would be nice to give feedback to the user as to why this is happening. though understand it could bloat the console especially if these errors are expected (i.e. a derived address that is non-existent). wonder if this is okay? or should it be under a flag? or just remove it altogether?

Reply:
Good point! There are a few places where we display warning messages already. Might be nice to make this configurable at some point (e.g. you can switch off warnings, only show in dev mode, etc.).